modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-azure.adoc

```diff
@@ -0,0 +1,183 @@
+= Run Teradata Jupyter Notebook Demos for VantageCloud Lake in Azure
+:experimental:
+:page-author: Daniel Herrera
+:page-email: daniel.herrera2@teradata.com
+:page-revdate: January 26th, 2024
+:description: Run Teradata Jupyter Notebook Demos for VantageCloud Lake in Azure
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, cloud computing, machine learning, sagemaker, vantagecloud, vantagecloud lake, lake
+:dir: vantagecloud-lake-demo-jupyter-azure
+:auxdir: vantagecloud-lake-demo-jupyter-sagemaker
+
+== Overview
+This quickstart details the process for running the https://github.com/Teradata/lake-demos[Teradata Jupyter Notebook Demos for Vantage Cloud Lake], on Microsoft Azure.
+
+== Prerequisites
+* Access to a Microsoft Azure account
+* https://quickstarts.teradata.com/getting-started-with-vantagecloud-lake.html[Access to a VantageCloud Lake environment]
```

Comment:
It might be just me, but when I click on "Access to a VantageCloud Lake environment" I expect to be able to get access to VantageCloud Lake environment. The link is just an intro to Lake. Maybe we can say this:

```suggestion
* Access to a VantageCloud Lake environment (https://quickstarts.teradata.com/getting-started-with-vantagecloud-lake.html[read introduction to VantageCloud Lake], https://www.teradata.com/platform/vantagecloud[request access to VantageCloud Lake])
```

Reply:
Agreed.
Also.  We have included the same link in both SageMaker and Vertex AI. 
https://quickstarts.teradata.com/vantagecloud-lake/vantagecloud-lake-demo-jupyter-sagemaker.html
https://quickstarts.teradata.com/vantagecloud-lake/vantagecloud-lake-demo-jupyter-google-cloud-vertex-ai.html
Also there is a similar verbiage in all the quickstarts, I'm changing to a verbiage that, for now, points the people to sales to request an account and also mention the guide. I will add this as a partial so we can just agree on this and then change it at once for all, and also we can change it in one go when the public trials open.